assets/text/api/v1-compat/auth/oss/basic-auth.sh

```diff
@@ -0,0 +1,15 @@
+#######################################
+# Use an InfluxDB 1.x compatible username
+# and password with Basic Authentication
+# to query the InfluxDB 1.x-compatibility API
+#######################################
+# Use default retention policy
+#######################################
+# Use the --user option with `--user <username>:<password>` syntax
+# or the `--user <username>` interactive syntax to ensure your credentials are
+# encoded in the header.
+#######################################
```

Comment:
```suggestion
#######################################

```

Something to think about – the content currently in these comments may be more readable outside of the code block as paragraph text.

Reply:
Debated that, but went with the comment since it's specific to curl